internal/testutil/acc/shared_resource.go

```diff
@@ -0,0 +1,48 @@
+package acc
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// SetupSharedResources must be called from TestMain test package in order to use ProjectIDExecution.
+func SetupSharedResources() {
+	sharedInfo.init = true
+}
+
+// CleanupSharedResources must be called from TestMain test package in order to use ProjectIDExecution.
+func CleanupSharedResources() {
+	if sharedInfo.projectID != "" {
+		fmt.Printf("Deleting execution project: %s, id: %s\n", sharedInfo.projectName, sharedInfo.projectID)
```

Comment:
how are we running tests? normally you want logs when running in verbose mode `-v`

Reply:
we use verbose flag:

TF_ACC=1 go test $(ACCTEST_PACKAGES) -run '$(ACCTEST_REGEX_RUN)' -skip '$(ACCTEST_REGEX_SKIP)' -v -parallel $(PARALLEL_GO_TEST) $(TESTARGS) -timeout $(ACCTEST_TIMEOUT) -ldflags="$(LINKER_FLAGS)"